README.md

```diff
@@ -263,3 +263,14 @@ For example:
 -from six.moves import StringIO
 -from six.moves.urllib.parse import quote_plus
 ```
+
+## Rewriting mock imports
+
+With `--py3-plus`, `reorder-python-imports` will also rewrite various `mock` imports:
+
+```diff
+- from mock import patch
++ from unittest.mock import patch
+```
+
+See `reorder_python_imports.py:MOCK_RENAMES` for a list.
```

Comment:
I could write a hook to populate this, or leave this out entirely

Reply:
I would just leave it out 🤷  -- we don't enumerate the `six` stuff and I think it's probably safe to assume "all of them"

Reply:
Sure